src/main/java/lv/ctco/javaschool/app/boundary/TripApi.java

```diff
@@ -110,5 +109,31 @@ public void createNewTrip(TripDto dto) {
 
         em.persist(trip);
     }
-}
 
+    @GET
+    @Path("/{id}/passengers")
+    @Produces("application/json")
+    @RolesAllowed({"ADMIN", "USER"})
+    public List<UserLoginDto> getTripPassengersByTripId(@PathParam("id") Long tripId) throws UserNotFoundException {
```

Comment:
API method which throws an exception is a bit weird. `UserNotFoundException` should be _not checked exception_. Please extend it from `RuntimeException`.